docs/core/configuration.md

```diff
@@ -512,15 +512,22 @@ timeout_commit = "1s"
 Note that in a successful round, the only timeout that we absolutely wait no
 matter what is `timeout_commit`.
 Here's a brief summary of the timeouts:
+<<<<<<< HEAD
 - `timeout_propose` = how long we wait for a proposal block before prevoting nil
 - `timeout_propose_delta` = how much  `timeout_propose` increases with each round
 - `timeout_prevote` = how long we wait after receiving +2/3 prevotes for
+=======
+
+- `timeout_propose` = how long a validator should wait for a proposal block before prevoting nil
+- `timeout_propose_delta` = how much `timeout_propose` increases with each round
+- `timeout_prevote` = how long a validator should wait after receiving +2/3 prevotes for
+>>>>>>> 3d1c36d1e (update language (#1263))
```

Comment:
There is a conflict here.

Reply:
Not sure what the conflict is?

Reply:
Check lines 515, 519 and 524.
These indicate that the section between 515-519  conflict with the lines 519-524.
Both sections describe the same parameters. You need to pick the correct/final version (probably the second) and remove the  <<< ==== >>> markers.

Reply:
Check what I did in the other back port branch. It had exactly the same problem.